<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Dell-PC" ignored="0" name="testMethodTwo" tests="3" failures="1" timestamp="2021-12-02T12:56:27 IST" time="19.97" errors="0">
  <testcase name="Tc_CheckBorrowingEstimate" time="8.126" classname="tc.TC_Calclulator">
    <failure type="java.lang.AssertionError" message="expected [$483,005] but found [$483,000]">
      <![CDATA[java.lang.AssertionError: expected [$483,005] but found [$483,000]
at tc.TC_Calclulator.ValidateCalculate(TC_Calclulator.java:57)
at tc.TC_Calclulator.Tc_CheckBorrowingEstimate(TC_Calclulator.java:15)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- Tc_CheckBorrowingEstimate -->
  <testcase name="Tc_LivingExpenseValidation" time="5.115" classname="tc.TC_Calclulator"/>
  <testcase name="Tc_VerifyStartOver" time="4.378" classname="tc.TC_Calclulator"/>
</testsuite> <!-- testMethodTwo -->
